client/executor/src/executor.rs

```diff
@@ -107,7 +105,7 @@ impl<H> WasmExecutorBuilder<H> {
 			onchain_heap_alloc_strategy: None,
 			offchain_heap_alloc_strategy: None,
 			max_runtime_instances: 2,
-			runtime_cache_size: 4,
+			runtime_cache_size: 8,
```

Comment:
Please revert

Reply:
I want to know why? 
I see the `8` value everywhere, but the new api set `4` as default value.

Reply:
I want to know why? 
I see the `8` value everywhere, but the new api set `4` as default value.